packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -257,7 +257,8 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
      */
     useEffect(() => {
         if (mapsindoorsSDKAvailable) {
-            const languageToUse = language ? language : navigator.language;
+            // Sets language to use. Priority: Prop -> App Config -> browser's default language.
+            const languageToUse = language ?? appConfig.appSettings.language ?? navigator.language;
```

Comment:
The assumption that there will always be an `appConfig` object here is causing the app to crash:

![image](https://github.com/user-attachments/assets/724ae701-daae-4795-92e0-f13e7a32636c)